README.md

```diff
@@ -4,6 +4,8 @@
 [![YouTube](https://img.shields.io/badge/Video-YouTube-red)](https://cortx.link/videos)
 [![Latest Release](https://img.shields.io/github/v/release/Seagate/cortx?label=Latest%20Release)](https://github.com/seagate/cortx/releases/latest)
 [![GitHub contributors](https://img.shields.io/github/contributors/Seagate/cortx)](https://github.com/Seagate/cortx/graphs/contributors/)
+[![SODA Eco project](https://img.shields.io/badge/SODA-ECO%20Project-9cf)](https://github.com/swatid-seagate/cortx/blob/main/doc/Soda-welcome-page.md)
```

Comment:
This is linking to the welcome page in your fork. It should just be a relative link so that it points correctly to the file in the main branch of the main repo.